std_msgs/Header header
sensor_msgs/Image img_ros_src
sensor_msgs/Image img_ros_res
BoundingBox[] bounding_boxes
